modules/haloRtdProvider.js

```diff
@@ -69,15 +70,12 @@ function paramOrDefault(param, defaultVal) {
  * @param {Object} rtdConfig
  */
 export function addRealTimeData(bidConfig, rtd, rtdConfig) {
-  if (!isPlainObject(config.getConfig().ortb2)) {
-    config.setConfig({ortb2: {}});
-  }
+  let ortb2 = config.getConfig().ortb2 || {};
```

Comment:
if you pass in the specific config to getConfig you want, it will only return that, instead of the entire configs which have been set.

`config.getConfig('ortb2');`

This is recommended because If for some reason `config.getConfig()` does not return an object, then `config.getConfig().ortb2` will throw an exception.

Reply:
thanks, fixed